integrationtest/TargetProjects/IntegrationTestApp.sln

```diff
@@ -0,0 +1,64 @@
+﻿
```

Comment:
Don't we already have a solution file the the integration test project?

Reply:
We have one including the validation tests. But now I wanted one with just the target project to run solution runs against.

Reply:
Do you agree with this? Or else we can add a new issue to clean up the integration test project as it is a bit of a mess right now :(

Reply:
I think that's good in any case